common/environment/build-style/cmake.sh

```diff
@@ -1,7 +1,10 @@
 if [ "$CHROOT_READY" ]; then
-	if [ "$pkgname" != cmake ]; then
-		hostmakedepends+=" cmake"
-	fi
+	# Ideally, everything should be built with cmake,
+	# problematic package will be built with cmake-bootstrap explicitly.
+	case "$(printf ' %s' $pkgname $hostmakedepends; echo ' ')" in
+	*" cmake-bootstrap "*|*" cmake "*) ;;
+	*) hostmakedepends+=" cmake"
```

Comment:
Is there actually a reason to check if cmake is already there? Pretty sure they will be deduplicated anyways and just checking for cmake-boostrap would avoid this subshell.

Reply:
Because I would like to allow special package to use `cmake` instead of `cmake-bootstrap`.
But it doesn't hurt to use `cmake-bootstrap` in all case I think